src/main/org/apache/tools/ant/listener/Log4jListener.java

```diff
@@ -142,21 +142,17 @@ public void messageLogged(final BuildEvent event) {
         final Logger log
             = Logger.getLogger(categoryObject.getClass().getName());
         switch (event.getPriority()) {
-            case Project.MSG_ERR:
-                log.error(event.getMessage());
-                break;
             case Project.MSG_WARN:
                 log.warn(event.getMessage());
                 break;
             case Project.MSG_INFO:
                 log.info(event.getMessage());
                 break;
             case Project.MSG_VERBOSE:
-                log.debug(event.getMessage());
-                break;
             case Project.MSG_DEBUG:
                 log.debug(event.getMessage());
                 break;
+            case Project.MSG_ERR:
```

Comment:
Personally I prefer the explicit case over a fallthrough to the default case.

Reply:
It is still explicit...

Reply:
not explicit enough for my taste :-)